### ISSUE TITLE
When I upgrade to 0.10.0 that bson is ^5.0.0, the ObjectId() on web throw error: Unsupported operation: Int64 accessor not supported by dart2js.

### ISSUE THREAD

**Issue by yangfanyu** [mongo-dart/mongo_dart#357]
I compared the code about ObjectId() in bson ^4.0.0 and bson ^5.0.0:

In bson ^4.0.0:

```dart

  static BsonBinary createId(int seconds, bool clientMode) {
    String getOctet(int value) {
      var res = value.toRadixString(16);
      while (res.length < 8) {
        res = '0$res';
      }
      return res;
    }

    if (clientMode) {
      var s = '${getOctet(seconds)}${getOctet(Statics.randomId)}'
          '${getOctet(Statics.nextIncrement)}';
      return BsonBinary.fromHexString(s);
    } else {
      return BsonBinary(12)
        ..writeInt(seconds, endianness: Endian.big)
        ..writeInt(Statics.randomId)
        ..writeInt(Statics.nextIncrement, endianness: Endian.big);
    }
  }


```

In bson ^5.0.0:

```dart

  /// creates a BsonBinary representation of this ObjectId with the given number
  /// of seconds sinceEpoch
  static BsonBinary createId(int seconds) => BsonBinary(12)
    ..writeInt(seconds, endianness: Endian.big)
    ..setIntExtended(_randomValue, 5)
    ..setIntExtended(_nextIncrement, 3, endianness: Endian.big);


```

then I change the code in bson ^5.0.0 to :

```dart

  /// creates a BsonBinary representation of this ObjectId with the given number
  /// of seconds sinceEpoch
  static BsonBinary createId(int seconds) => BsonBinary(12)
        ..writeInt(seconds, endianness: Endian.big)
        ..writeInt(Statics.randomId)
        ..writeInt(Statics.nextIncrement, endianness: Endian.big);


```

every thing is ok.

best regards.

**Comment by giorgiofran** [comment]
Uploaded a fix. Can you please give me a feedback? Thanks in advance!

**Comment by yangfanyu** [comment]
@giorgiofran Thanks, this problem is fixed in `bson 5.0.3`
